client/gutenboarding/gutenboard.tsx

```diff
@@ -39,30 +37,48 @@ export function Gutenboard() {
 		updateIsEditorSidebarOpened( ! isEditorSidebarOpened );
 	}
 
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	return (
-		<div className="gutenboarding">
-			<Header
-				isEditorSidebarOpened={ isEditorSidebarOpened }
-				toggleGeneralSidebar={ toggleGeneralSidebar }
-			/>
+		<div className="gutenboarding block-editor__container">
 			<SlotFillProvider>
 				<DropZoneProvider>
-					<BlockEditorProvider value={ [ onboardingBlock ] } settings={ { templateLock: 'all' } }>
-						<div className="gutenboarding__block-editor">
-							<BlockEditorKeyboardShortcuts />
-
-							<SettingsSidebar isActive={ isEditorSidebarOpened } />
-							<Sidebar.Slot />
-							<WritingFlow>
-								<ObserveTyping>
-									<BlockList />
-								</ObserveTyping>
-							</WritingFlow>
-						</div>
-						<Popover.Slot />
-					</BlockEditorProvider>
+					<div
+						className={ classnames( 'edit-post-layout', {
+							'is-sidebar-opened': isEditorSidebarOpened,
+						} ) }
+					>
+						<Header
+							isEditorSidebarOpened={ isEditorSidebarOpened }
+							toggleGeneralSidebar={ toggleGeneralSidebar }
+						/>
+						<BlockEditorProvider value={ [ onboardingBlock ] } settings={ { templateLock: 'all' } }>
+							<div className="edit-post-layout__content">
```

Comment:
If we're introducing this in order to replicate styling more closely, maybe we should ditch the `.gutenboarding__block-editor` div on the line below? I remember I was copying a bunch of styling from upstream's `.edit-post-layout__content` to our `.gutenboarding__block-editor` selector; can we merge it into the other one (to keep it more obvious which parts are coming from upstream's `.edit-post-layout__content`.)

Reply:
Valid point! I removed the wrapper. Its main job was to provide `box-sizing: border-box;` styles but that's now handled by `.edit-post-layout` so we're good.